contrib/raftexample/httpapi.go

```diff
@@ -24,13 +24,7 @@ import (
 	"github.com/coreos/etcd/raft/raftpb"
 )
 
-// Handler for a http based key-value store backed by raft
-type httpKVAPI struct {
-	store       *kvstore
-	confChangeC chan<- raftpb.ConfChange
-}
-
-func (h *httpKVAPI) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+func (rc *raftNode) ServeHTTP(w http.ResponseWriter, r *http.Request) {
```

Comment:
all of this should still be `httpKVAPI`; the http layer is meant to be separate from the raft layer